t/critic.t

```diff
@@ -8,10 +8,14 @@
 use 5.10.1;
 use strict;
 use warnings;
-use lib qw(. lib local/lib/perl5);
+use lib qw(. lib local/lib/perl5 t);
+
+use Support::Files;
 use Test::More;
 
+my @testitems = (@Support::Files::testitems, @Support::Files::test_files);
```

Comment:
This now excludes files in the scripts directory, which should be covered by linting.

Perhaps `@Support::Files::testitems` should be extended to include all files under scripts with a first line ending in ` perl`?